### PR TITLE
Bump Apps' container tags

### PR DESCRIPTION
**Pull request by charagender**
Prowlarr image was really old as mentioned in Issue #19 so this updates it to use the latest image available

### CODE REVIEW

**Comment by charagender** [comment]
After going through more apps, I noticed a lot of them are static versions and would have to be updated manually by going into their `docker-compose.yml` files and bumping their versions. Should they be updated to use `latest` instead?

File: Apps/Prowlarr/docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
       PGID: "1000"
       PUID: "1000"
       TZ: Europe/London
-    image: linuxserver/prowlarr:0.4.5-nightly
+    image: lscr.io/linuxserver/prowlarr:latest
```
Comment (tigerinus) [inline Apps/Prowlarr/docker-compose.yml:8]: Please use specific tag instead of `latest`. Using `latest` adds version uncertainty.

**Review by tigerinus — CHANGES_REQUESTED** [review]

File: Apps/2FAuth/docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: 2fauth
 services:
   2fauth:
-    image: 2fauth/2fauth:3.4.2
+    image: 2fauth/2fauth:4.0.2
```
Comment (tigerinus) [inline Apps/2FAuth/docker-compose.yml:4]: That's a change in major version. Have you tested this version against existing configurations in the compose yaml (i.e. env, memory, etc.)?

If not, it's simpler to just update its minor/build version for now.

File: Apps/Grafana/docker-compose.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 name: grafana
 services:
   grafana:
-    image: grafana/grafana:8.4.5
+    image: grafana/grafana:9.5.2
```
Comment (tigerinus) [inline Apps/Grafana/docker-compose.yml:4]: See the comments about major version change above.

File: Apps/Nextcloud/docker-compose.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 name: nextcloud
 services:
   nextcloud:
-    image: nextcloud:25.0.4
+    image: nextcloud:26.0.1
```
Comment (tigerinus) [inline Apps/Nextcloud/docker-compose.yml:4]: See the comments about major version change above.

File: Apps/Node-RED/docker-compose.yml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 name: node-red
 services:
   node-red:
-    image: nodered/node-red:2.2.2-12
+    image: nodered/node-red:3.0.2-18
```
Comment (tigerinus) [inline Apps/Node-RED/docker-compose.yml:4]: See the comments about major version change above.

File: Apps/Prowlarr/docker-compose.yml [hunk 6]
```diff
@@ -5,7 +5,7 @@ services:
       PGID: "1000"
       PUID: "1000"
       TZ: Europe/London
-    image: linuxserver/prowlarr:0.4.5-nightly
+    image: lscr.io/linuxserver/prowlarr:1.4.1
```
Comment (tigerinus) [inline Apps/Prowlarr/docker-compose.yml:8]: See the comments about major version change above.

File: Apps/Sabnzbd/docker-compose.yml [hunk 7]
```diff
@@ -5,7 +5,7 @@ services:
       PGID: "1000"
       PUID: "1000"
       TZ: Europe/London
-    image: linuxserver/sabnzbd:3.7.2
+    image: linuxserver/sabnzbd:4.0.1
```
Comment (tigerinus) [inline Apps/Sabnzbd/docker-compose.yml:8]: See the comments about major version change above.

File: Apps/Transmission/docker-compose.yml [hunk 8]
```diff
@@ -9,7 +9,7 @@ services:
       TRANSMISSION_WEB_HOME: /transmission-web-control/
       TZ: Europe/London
       USER: casaos
-    image: lscr.io/linuxserver/transmission:version-3.00-r5
+    image: lscr.io/linuxserver/transmission:4.0.3
```
Comment (tigerinus) [inline Apps/Transmission/docker-compose.yml:12]: See the comments about major version change above.

**Comment by tigerinus** [comment]
@charagender I am so happy that you took the action and take over updating app tags! Huge thanks!

There is just one issue about all major version changes. Please see my comments in the code.

**Review by tigerinus — CHANGES_REQUESTED** [review]

**Comment by charagender** [comment]
> @charagender I am so happy that you took the action and take over updating app tags! Huge thanks!
> 
> There is just one issue about all major version changes. Please see my comments in the code.

That's what I was about to do actually! That's why I hadn't yet done a request for comment. I'm just trying to install Debian on my computer now to test all the changes, because I don't have a way of controlling the computer I have CasaOS installed on anymore

**Comment by charagender** [comment]
@tigerinus yes! it's just that I haven't had much time to dedicate to things other than studying recently because of uni exams, though I think I'll have time to test things this weekend

**Comment by tigerinus** [comment]
This work is mostly covered in https://github.com/IceWhaleTech/CasaOS-AppStore/pull/176

Thanks again @charagender for starting it. Looking forward to more contributions from you!

**Comment by charagender** [comment]
alright, thank you! I hope I'm able to! turns out uni is harder than I thought, so that's my priority right now

**Pull request closed without merging**